webapp-django/crashstats/topcrashers/views.py

```diff
@@ -213,6 +213,15 @@ def topcrashers(request, days=None, possible_days=None, default_context=None):
                     request.build_absolute_uri(), urlquote(pv['version'])
                 )
                 return redirect(url)
+        if context['active_versions'][product]:
+            # Not a single version was featured, but there were active
+            # versions. In this case, use the first available
+            # *active* version.
+            for pv in context['active_versions'][product]:
+                url = '%s&version=%s' % (
+                    request.build_absolute_uri(), urlquote(pv['version'])
+                )
+                return redirect(url)
```

Comment:
There is a bit of repeated code here but I think it's really really easy to read and there isn't much actual business logic that is repeated.